filter_test.go

```diff
@@ -21,10 +21,10 @@ func TestFuzzyFilter(t *testing.T) {
 		{"this is a Test to test the fUzzy filter", "TU", true},  // case sensitivity
 		{"this is a test to test the fuzzy filter", "Tu", false}, // case sensitivity
 		{"this is a test to Test the fuzzy filter", "TU", false}, // case sensitivity
-		{"日本語は難しいです", "難", true}, // kanji
-		{"あ、日本語は難しいですよ", "あい", true}, // hiragana
-		{"パソコンは遅いですネ", "ソネ", true}, // katakana
-		{"🚴🏻 abcd efgh", "🚴🏻e", true}, // unicode
+		{"日本語は難しいです", "難", true},                                 // kanji
+		{"あ、日本語は難しいですよ", "あい", true},                             // hiragana
+		{"パソコンは遅いですネ", "ソネ", true},                               // katakana
+		{"🚴🏻 abcd efgh", "🚴🏻e", true},                            // unicode
 	}
```

Comment:
please add test for empty string. and test matches first/last latter.

```
This is a test to Test the fuzzy filteR
```
then `TTR`